packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts

```diff
@@ -46,6 +46,7 @@ const createOptions = <Method extends Exclude<keyof Client, 'clientConfig'>>(
 const basketsEndpoint = '/checkout/shopper-baskets/'
 const BASKET_ID = 'basket_id'
 const getBasketOptions = createOptions<'getBasket'>(undefined, {})
+const emptyBasket: Basket = {}
```

Comment:
This is a constant that you never want people to mutate it...

```suggestion
const EMPTY_BASKET: Basket = {}
```

Reply:
nit, i see only 1 test uses this, why don't we declare it inside that test?